pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[build-system]
+requires = [ "setuptools>=61", "wheel", "setuptools-git-versioning>=2.0,<3", ]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools-git-versioning]
+enabled = true # Allows setuptools-git-versioning to pull version from git state
+
+[tool.setuptools]
+py-modules = []
+
+[project]
+name = "model-interpeter"
+readme = "README.md"
+license = {'text' = "Proprietary"}
+authors = [
+    {name = "LV GI Data Science Team", email = "#DataSciencePackages@lv.co.uk"},
+]
+description  = "this package contains functionality for reporting on how your ML model is making decisions"
+requires-python = ">= 3.9"
+dynamic = ["version"]
+
+dependencies = [
+    "numpy",
+    "numba",
+    "shap",
+]
+
+[project.optional-dependencies]
+dev = [
+    "xgboost",
+    "pytest",
+    "pytest-mock",
+    "pytest-cov",
+    "pytest-benchmark",
+    "bandit",
```

Comment:
Do you need bandit in here?

Reply:
Also, consider using xgboost-cpu so that you don't need all the GPU drivers

Reply:
good catch, general env updates here
https://github.com/lvgig/model_interpreter/pull/2/commits

switched xgboost here https://github.com/lvgig/model_interpreter/pull/2/commits/5ed23c71d88205a44e83368f35c8d75079c9be3f

Reply:
Nice. Would like to come back to dependencies at some stage - numba is in here, but that drags in all of llvm and is quite beefy. Out of scope though. Thanks for these changes.